### PR TITLE
Adding optional login after install and setup

### PR DESCRIPTION
**Pull request by bpronan**
This is needed to simplify our quick start setup.

### CODE REVIEW

File: install-git-xet.sh [hunk 1]
```diff
@@ -41,5 +85,8 @@ case "$ARCH" in
     echo "Unsupported architecture: $ARCH; please see available installation options at https://xethub.com/assets/docs/getting-started/install"
 esac
 
+
 install $DIST
```
Comment (assafvayner) [inline install-git-xet.sh:89]: if install fails, should not attempt to login, exit early

File: install-git-xet.sh [hunk 2]
```diff
@@ -13,11 +13,55 @@ function install () {
...
+    return 0
+  fi
+
+  if [ -z "$host" ]
+  then
+    echo "Authenticating with XetHub.com..."
+    git xet login -u $username -e $email -p $token ; local CODE=$?
+  else
+    echo "Authenticating with ${host}..."
+    git xet login -u $username -e $email -p $token --host $host; local CODE=$?
```
Comment (assafvayner) [inline install-git-xet.sh:33]: tiny nit on semicolon placement, put a space after $host and the semicolon

**Review by assafvayner — APPROVED** [review]
please address the 2 nit picky comments, otherwise good to go in

File: install-git-xet.sh (hunk 1 above)
Comment (bpronan) [reply install-git-xet.sh:104]: Good catch. Thanks!

**Pull request merged**